modApiServer/src/org/aion/api/server/types/TxRecptLg.java

```diff
@@ -56,7 +52,7 @@
         this.blockHash = b == null ? null : TypeConverter.toJsonHex(b.getHash());
         this.transactionIndex = (b == null || txIndex == null) ? null : TypeConverter.toJsonHex(txIndex);
         this.transactionHash = TypeConverter.toJsonHex(tx.getHash());
-        this.address = (tx == null || tx.getTo() == null) ? "" : TypeConverter.toJsonHex(tx.getTo().toBytes());
+        this.address = TypeConverter.toJsonHex(logInfo.getAddress().toString());
```

Comment:
Is it guaranteed that `logInfo` and `logInfo.getAddress()` are not null?

Reply:
it is guaranteed